lib/config.js

```diff
@@ -57,20 +54,14 @@ var saveConfig = function(config) {
   fs.writeFileSync(CONFIG_PATH, JSON.stringify(config, null, 2), 'utf8');
 };
 
-var getGlobalConfig = function() {
+var getKernelPort = function() {
   var config = null;
-
   if (fs.existsSync(GLOBAL_CONFIG)) {
     config = fs.readFileSync(GLOBAL_CONFIG);
     config = JSON.parse(config);
-    config.kernel = config.kernel || {};
-  } else {
-    throw new Error('No nscale-client config file found.\n' +
-        '  Expected to see one at ' + GLOBAL_CONFIG + '\n' +
-        '  Have you started nscale (I.E. nscale start)?');
   }
 
-  return config;
+  return config.kernel.port || DEFAULT_PORT;
```

Comment:
Yes I can fix this, however the kernel itself does depend on the kernel section existing in the config